src/client/src/apps/MainRoute/widgets/spotTile/epics/rfqEpics.ts

```diff
@@ -78,20 +90,38 @@ export const rfqReceivedEpic: ApplicationEpic = action$ =>
   action$.pipe(
     ofType<Action, RfqReceivedActionType>(TILE_ACTION_TYPES.RFQ_RECEIVED),
     mergeMap(action => {
+      const { currencyPair } = action.payload
       const cancel$ = action$.pipe(
         ofType<Action, RfqReceivedTimerCancellableType>(
           TILE_ACTION_TYPES.RFQ_REJECT,
-          TILE_ACTION_TYPES.RFQ_EXPIRED,
           TILE_ACTION_TYPES.RFQ_RESET,
         ),
-        filter(
-          cancelAction =>
-            cancelAction.payload.currencyPair.symbol === action.payload.currencyPair.symbol,
-        ),
+        filter(cancelAction => cancelAction.payload.currencyPair.symbol === currencyPair.symbol),
       )
 
       return timer(action.payload.timeout + 1000).pipe(
-        map(() => rfqExpired({ currencyPair: action.payload.currencyPair })),
+        mergeMap(() =>
```

Comment:
Potentially you could use `concat` to make this slightly clearer (I think) because the timers would then be at a similar level...

```
return concat(
  timer(action.payload.timeout + 1000).pipe(map(() => /* expired */)),
  timer(IDLE_TIME_MS).pipe(mergeMap(() => /* reset */))
).pipe(takeUntil(cancel$))
```